src/nodes.coffee

```diff
@@ -1431,9 +1430,8 @@ exports.Param = class Param extends Base
     return @reference if @reference
     node = @name
     if node.this
-      node = node.properties[0].name
-      if node.value.reserved
-        node = new Literal o.scope.freeVariable node.value
+      name = "at_#{node.properties[0].name.value}"
```

Comment:
Seems like it. The [`utility` function](https://github.com/lydell/coffee-script/blob/8ab15d737225acf3d0c2e9d575225df6a470971a/src/nodes.coffee#L2262) should register it.